e2e/tests/cluster_restart.robot

```diff
@@ -9,6 +9,7 @@ Test Teardown    Cleanup test resources
 
 *** Variables ***
 ${LOOP_COUNT}    1
+${LATENCY_IN_MS}    0
```

Comment:
Make the naming meaningful like `CONTROL_PLANE_NODE_NETWORK_LATENCY_IN_MS`

The same for `${LOOP_COUNT}`.

Reply:
Refined.